MobSF/utils.py

```diff
@@ -56,6 +56,10 @@ def upstream_proxy(flaw_type):
 
 def api_key():
     """Print REST API Key"""
+
+    if os.environ.get('API_KEY'):

```

Comment:
I suggest changing the environment variable to something that is not common or conflicting with other apps. Something like `MOBSF_API_KEY` should be fine.

Reply:
Also when API key is read from env variable, print a message like

`API Key read from environment variable` just to avoid confusion.

Reply:
fixed, thanks!